bin/casper_job_submit.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -l
+#PBS -N SAMURAI
+#PBS -A NTDD0004
+#PBS -l select=1:ncpus=36:ompthreads=1:mem=700GB:ngpus=1
+#PBS -l gpu_type=v100
+#PBS -q casper
+#PBS -l walltime=00:30:00
+#PBS -j oe
+#PBS -k eod
+ 
+cd $PBS_O_WORKDIR
+
+##################
+# Build the code #
+##################
+cd ..
+./bin/ncar_build.sh gpu
```

Comment:
@sjsprecious I think we need more flexibility here.  We want to be able to run both the CPU and GPU versions of the code easily.  Just hard coding this for GPU execution is not what we want.

Reply:
Thanks @johnmauff . A user can remove the `gpu` option in this script to build a CPU case instead. I just use a GPU case as an example. Note that this is a PBS script so I don't think it will take any argument to switch between CPU and GPU easily. Correct me if I am wrong here.

Reply:
Maybe a workaround would be to have GPU and CPU versions of the run scripts.  When we set up our CI/CD we will want the ability easily check on both CPU and GPU platforms

Reply:
Thanks @johnmauff for the suggestion. That will work. Do you want me to generate separate run scripts for V100, A100, etc as well? Or we should ask the user to change the architecture based on their desires?

Reply:
I think, for now, just a single GPU script is fine.  We may need to do something a bit more sophisticated depending on the platforms in which we apply CI/CD.